lib/worker.go

```diff
@@ -495,7 +501,7 @@ func (m *masterClient) CheckMasterTimeout(ctx context.Context, clock clock.Clock
 	if sinceLastAcked > 2*m.timeoutConfig.workerHeartbeatInterval &&
 		sinceLastAcked < m.timeoutConfig.workerTimeoutDuration {
```

Comment:
we should use one configure parameter here instead of two, such as  the wokerTimeDuration  = 2* workerHeartbeatInterval + 3.

Reply:
Yes, too many parameters make it difficult for debugging, we need to reduce the complexity, but seems only assign `wokerTimeDuration = 2* workerHeartbeatInterval + 3` doesn't reduce the complexity (The check condition still contains these variables).

Reply:
addressed, ptal @nongfushanquan